docs/user-guide/how-to.md

```diff
@@ -186,6 +187,15 @@ permissions. To understand why an object is not being propagated to a namespace,
 use `kubectl hns describe <ns>`, where `<ns>` is either the source (ancestor) or
 destination (descendant) namespace.
 
+<a name="use-hrq"/>
+
+### Limit Resources over parent namespaces
+
+HNC has an object called HirarchicalResourceQuota that is similar to ResourceQuota
+but across all namespace tree. It allows you to distribute tour resources between
+teams, and those teams can distribute their resources between their subteams...
```

Comment:
"between their subteams..." -> "flexibly between their namespaces, without worrying about the exact usage in any one namespace."

Reply:
same as i answered in previous note,
it sounds easier to understand, but my point was to not make them think that "hrq can work perfectly across your subnamespaces and you dont need to worry about their sum".
i was trying to explain that each team has their own permissions, and amount of reasources, and they can distribute their own resources as they want between their sub teams without involving anyone else.

Reply:
It might be worth mentioning that if this is what people want, you can use `RoleBindings` to give team admins the permission to create regular `ResourceQuota` objects, which will solve that problem if the goal is to limit consumption in the "leaf" namespaces.